.github/workflows/release.yaml

```diff
@@ -94,6 +94,12 @@ jobs:
                 --build-arg TAG=$TAG \
                 ${dir}
           done
+
+  scan:
```

Comment:
You need to consider where is the working directory, because on the `build` job it's specified.
https://github.com/cybozu/ubuntu-base/blob/7afa25a582fbbc5ccd98cfeb620d7fa547e95977/.github/workflows/release.yaml#L16-L18